src/cryptonote_core/tx_pool.cpp

```diff
@@ -59,27 +59,31 @@ namespace cryptonote
   namespace
   {
     /*! The Dandelion++ has formula for calculating the average embargo timeout:
-                          (-k*(k-1)*hop)/(2*log(1-ep))
+                          (-k*(k-1)*hop)/(2*ln(1-ep))
         where k is the number of hops before this node and ep is the probability
         that one of the k hops hits their embargo timer, and hop is the average
         time taken between hops. So decreasing ep will make it more probable
         that "this" node is the first to expire the embargo timer. Increasing k
         will increase the number of nodes that will be "hidden" as a prior
         recipient of the tx.
 
-        As example, k=5 and ep=0.1 means "this" embargo timer has a 90%
-        probability of being the first to expire amongst 5 nodes that saw the
+        NOTE: The paper says `2*log(1-ep)`, however if you read the explanation
+        in b.5 it is clear they meant `ln`.
+
+        As example, k=10 and ep=0.1 means "this" embargo timer has a 90%
+        probability of being the first to expire amongst 10 nodes that saw the
```

Comment:
I don't think this is correct, it means the tx has a 90% probability of reaching k hops before _one of the nodes_ timer fires not 90% probability of the last one firing

Reply:
Fixed.